### PR TITLE
feat(#12): Profile view

### LINKED ISSUE #12: Profile view

### PR DESCRIPTION
**Pull request by MassimoFerro**
This view should display some user info that are public :
Screenshot 2021-12-29 at 02-54-56 Profile - TimeLife
**This branch contain work from the 35-albumlistview branch**

Resolves #12

### CODE REVIEW

**Review by Narekushia — COMMENTED** [review]
Here is my review : 
- I think you could use the `route()` syntax in Vue,
  - More details in this comment : https://github.com/HE-Arc/TimeLife/pull/36#pullrequestreview-840576547
  - For example, you could use it in `Menu.vue` Line 12 and 13 for the `href` :wink:
- It's kind of bizarre design vise because the description box and the albums preview have a drop shadow, but not the user avatar ? I don't know which is better : Everything with a shadow or nothing 🤔 
- Apart from that, it's all good ! I will approve when these changes and the one in #37 are adressed

File: resources/js/Pages/Profile.vue [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+.columns {
+    padding: 2rem;
+}
+
+.box {
+    margin-top: 1rem;
+    white-space: pre-wrap;
+    box-shadow: none;
+}
```
Comment (Narekushia) [inline resources/js/Pages/Profile.vue]:
```suggestion
    box-shadow: none;
    border: 1px solid rgba(0,0,0,.15);
}
```
Now that the box doesn't have a shadow anymore, the contrast between the box and the background is lacking.
I suggest here a change that add a thin border around the box, like this :
firefox_2022-01-03_19-22-22
Before, it was like that :
firefox_2022-01-03_19-22-32

Feel free to reject/change this suggestion if you have a better idea.

**Review by Narekushia — COMMENTED** [review]
Everything is pretty much ok now 👍 ✅ Good job !

I just suggested a change to add a border around the description box.

Don't forget to pull the last change from #35 and I will accept this PR ! :tada: :tada:

**Comment by MassimoFerro** [comment]
Thanks for your review, now everything should be ok.

**Review by Narekushia — APPROVED** [review]
:tada::tada: Everything is OK 👍 Good Job :tada::tada:

**Pull request merged**